src/python/grapl-model-plugin-deployer/src/grapl_model_plugin_deployer.py

```diff
@@ -75,6 +75,7 @@
     )["SecretString"]
 
 app = Chalice(app_name="model-plugin-deployer")
+app.debug = True
```

Comment:
what does this do?

Reply:
Sets up debug logging, but I'll remove this - I'll add it back in a separate PR where it's env-based.